dashboards/component/chart/serializers.py

```diff
@@ -180,8 +183,7 @@ class Meta(PlotlyChartSerializerMixin.Meta, BaseChartSerializer.Meta):
     class Media:
         js = ("dashboards/vendor/js/plotly.min.js",)
 
-
-class ChartSerializer(ModelDataMixin, PlotlyChartSerializer):
+class ChartSerializer(ModelDataMixin, PlotlyChartSerializer, FilterComponent):
```

Comment:
`FilterComponent` object should be passed in when the Serializer object is called, not inherited from, see https://github.com/wildfish/django-dashboards/blob/main/dashboards/component/base.py#L118

Reply:
okay